From: Reinhard Tartler <siretart@tauware.de>
Subject: in case of crash, suggest installing package mplayer-dbgsym for debugging symbols
Forwarded: not-needed

--- a/mplayer.c
+++ b/mplayer.c
@@ -825,7 +825,10 @@
         case SIGSEGV:
             mp_msg(MSGT_CPLAYER, MSGL_FATAL, MSGTR_Exit_SIGSEGV_SIGFPE);
         default:
-            mp_msg(MSGT_CPLAYER, MSGL_FATAL, MSGTR_Exit_SIGCRASH);
+            mp_msg(MSGT_CPLAYER, MSGL_FATAL,
+                  " [ This binary of MPlayer in Debian is currently compiled with\n"
+                  "   '--enable-debug'; the debugging symbols are in the package\n"
+                  "   'mplayer-dbgsym'.]\n");
 #ifdef CONFIG_CRASH_DEBUG
             if (crash_debug) {
                 int gdb_pid;
